data/json/professions.json

```diff
@@ -5231,8 +5240,7 @@
           { "item": "bandages", "count": 3 },
           "stethoscope",
           "scissors",
-          "syringe",
-          "morphine"
+          "adrenaline_injector"
```

Comment:
They did explicitly say the Paramedic having morphine is chill if you wanted to add that back on top of their epipen https://discord.com/channels/598523535169945603/598523535169945607/1131880091996147712

Reply:
oh dang, I think I axed that because I started adding med supplies before noticing the medkit was there. back in it goes!